grails-app/migrations/join_dets_and_rxrs.groovy

```diff
@@ -272,4 +272,57 @@ databaseChangeLog = {
                    viewName: 'invalid_detection'
         )
     }
+
+    changeSet(author: "jburgess", id: "1430268900000-09") {
+        createView("""
+            SELECT
+              station_station_name AS station,
+              installation_name AS installation,
+              rxr_project_name AS project,
+              aatams.scramblepoint(st_makepoint(station_longitude, station_latitude)) AS public_location,
+              st_x(aatams.scramblepoint(st_makepoint(station_longitude, station_latitude))) AS public_lon,
+              st_y(aatams.scramblepoint(st_makepoint(station_longitude, station_latitude))) AS public_lat,
+              ('http://localhost:8080/aatams/installationStation/show/'::text || station_id) AS installation_station_url,
+              ('http://localhost:8080/aatams/detection/lISt?filter.receiverDeployment.station.in=name&filter.receiverDeployment.station.in='::text || (station_station_name)::text) AS detection_download_url,
+              count(*) AS detection_count,
+              ((log((GREATEST(count(*), (1)::bigint))::double precISion) / log(((SELECT max(t.detection_count) AS max FROM (
+                SELECT
+                  station_station_name,
+                  count(station_station_name) AS detection_count
+                FROM aatams.detection_view GROUP BY station_station_name
+                ) t))::double precISion)) * (10)::double precISion) AS relative_detection_count,
```

Comment:
The capitalisation in 'precision' is cray-cray. Is that a real thing, or a copy-and-paste error?